techniques/failures/F85.html

```diff
@@ -133,38 +129,38 @@ <h3>Example 2: Setting focus to the document after dismissing a menu embedded on
 
 
 <section id="related">
-      <h2>Related Techniques</h2>
+      <h2>関連テクニック</h2>
       <ul>
         <li><a href="../client-side-script/SCR26">SCR26: Inserting dynamic content into the Document Object Model immediately following its trigger element</a></li>
       </ul>
     </section>
 <section id="tests">
-      <h2>Tests</h2>
+      <h2>検証</h2>
       <section class="procedure" id="procedure">
-        <h3>Procedure</h3>
-        <p>For each menu or dialog on a web page that is opened via a trigger control:</p>
+        <h3>手順</h3>
+        <p>トリガーコントロールによって開かれるウェブページ上の各メニュー又はダイアログに対して:</p>
         <ol>
           <li>
-            <p>Activate the trigger control via the keyboard.</p>
+            <p>キーボードでトリガーコントロールをアクティブにする。</p>
             <ul>
-              <li>Check whether focus has been set to the menu, dialog, or a logical focusable descendent of the widget.</li>
-              <li>If not, check whether moving the focus forward once in the sequential navigation order puts focus in the menu or dialog.</li>
+              <li>フォーカスがメニュー、ダイアログ、又はウィジェットの論理的にフォーカス可能な子孫に設定されているかどうかをチェックする。</li>
+              <li>そうでない場合は、シーケンシャルナビゲーション順序でフォーカスを一つ前に移動したときに、メニュー又はダイアログにフォーカスが設定されるかどうかをチェックする。</li>
```

Comment:
sequential navigation order の訳を揃えました。
```suggestion
              <li>そうでない場合は、シーケンシャルなナビゲーション順序でフォーカスを一つ前に移動したときに、メニュー又はダイアログにフォーカスが設定されるかどうかをチェックする。</li>
```